ci/Dockerfile-envoy-distroless

```diff
@@ -7,4 +7,4 @@ ADD linux/amd64/build_release${ENVOY_BINARY_SUFFIX}/* /usr/local/bin/
 
 EXPOSE 10000
 
-CMD ["envoy", "-c", "/etc/envoy/envoy.yaml"]
+ENTRYPOINT ["envoy", "-c", "/etc/envoy/envoy.yaml"]
```

Comment:
Shall we have both entrypoint and cmd like other images?

Reply:
i think its not necessary - but open to suggestion as to how do it - we cant add shell files as its distroless

this is not ideal, as it hardcodes the envoy config path - but it at least allows additional arguments to be passed through

Reply:
nm - i figured out how to use both and make the config path configurable